### ISSUE TITLE
Auth0::Api::V2::Actions#actions has prohibited parameters

### ISSUE THREAD

**Issue by rapito** [auth0/ruby-auth0#477]
### Checklist

- [X] I have looked into the [Readme](https://github.com/auth0/ruby-auth0#readme) and [Examples](https://github.com/auth0/ruby-auth0/blob/master/EXAMPLES.md), and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://www.rubydoc.info/gems/auth0) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/ruby-auth0/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

Cannot invoke main actions endpoint from management api using the library.
Here's the doc indicating the parameters: https://auth0.com/docs/api/management/beta/v2/get-actions
Here's the method signature: 
```rb
def actions(trigger_id, action_name, deployed: nil, per_page: nil, page: nil, installed: nil)
# ...
end 
```

### Reproduction

Executing this causes an error
- run: `Auth0Client.new(credentials).actions`
-error: `wrong number of arguments (given 0, expected 2) (ArgumentError)`

Passing any parameters results in:
```rb
 {"statusCode":400,"error":"Bad Request","message":"Query validation error: 'Additional properties not allowed: action_name,trigger_id'.","errorCode":"invalid_query_string"} (Auth0::BadRequest)

```

### ruby-auth0 version

5.13.0

### Ruby version

3.0.5

**Comment by stevehobbsdev** [comment]
Thanks for the report, I agree the API here in the SDK is a little broken. Following up with you on [the PR](https://github.com/auth0/ruby-auth0/pull/478) 👍🏻

**Comment by stevehobbsdev** [comment]
This has been resolved in #478, will be cutting a release hopefully later this week.
